lib/iex/lib/iex/helpers.ex

```diff
@@ -194,11 +198,57 @@ defmodule IEx.Helpers do
   end
 
   @doc """
+  Opens the current prying location.
+
+  This command only works inside a pry session started manually
+  via `IEx.pry` or a breakpoint set via `IEx.break/1`. Calling
+  this function during a regular `IEx` session will print an error.
+
+  Keep in mind the `open` location may not exist when prying
```

Comment:
If `open` here is meant as the function `open/0`, arity needed as well for self-linking, right?